File: ecommerce_store.py
import datetime

available_product = [ 'Bread' ,'videoGame' , 'Water' ]
available_product_price = [200 , 500, 100]
discount = 7.5 / 100
vat = 17.58 / 100
total = 0
sub_total = 0
bill_total = 0
total_vat = 0





def getcustomer_name():
	customer_name = input(" What is the customer's Name ")

def getavailable_product():
	for product in available_product:
		response = " yes"
	customer_product = input(" What did the user buy ")
	if (resons.equalsIgnoreCase(customer_product)):
			return customer_product

def getproduct_quantity(available_product):
	 product_quantity = int(input(" How many pieces " ))
				
for all_product_quantity in range(available_product_price):
		print( all_product_quantity)
			


def getcashier_name():
	cashier_name = input("Enter cashier Name")


def getstore_product_calculations(available_product_price , product_quantity ):
	 total =  available_product_price * product_quantity 
	 sub_total = total - (discount *   product_quantity) 
	 total_vat =  vat + (discount *  product_quantity)
	 bill_total = ( available_product_price * product_quantity ) +  			total_vat

print( " Receipt: ")
print( " =======================================================")	
print(     "  SEMICOLON STORES   ")
print(   "  MAIN BRANCH  ")  
print(  "Location: 312, HERBERT MACAULAY WAY, SABO YABA, LAGOS")
print( "  Date : " ,  datetime )
print( " Cashier : ",  getcashier_name)
print( "customer : ", getcustomer_name)
print("===========================================================")
print("  ITEM          QTY          PRICE          TOTAL(NGN) ")
print("------------------------------------------------------------")
print("%s %s%n %s%n  %s%n ",getavailable_product, getproduct_quantity, getproduct_quantity ,getstore_product_calculations )
print(" Sub Total :",(discount *   product_quantity))
print("Discount   -%2.f : " , sub_total )
print(" VAT +%2.f  : " ,  vat + (discount *  product_quantity) )
print("=============================================================")
print(" Bill Total: ", bill_total)
print("==============================================================")
print("THIS IS NOT AN RECEIPT, KINDLY PAY ",   getstore_product_calculations)
print("===============================================================")



def main():
	getcustomer_name()
	getavailable_product()
	getproduct_quantity(available_product)
	getcashier_name()
	getstore_product_calculations(available_product_price )




 
	
			

File: ecommercestore.js
function getcustomerName(){
let customerName = String("Enter customer's Name ")
return customerName;
}



function getcashierName(){
let cashierName = String("Enter customer's Name ")
return cashierName;
}



function getproductName(){
console.log("\n Available product : , Videogame ,  Water ,  Meat , Bread ")
let productName = String(" What do you want " )
return productName;
}


function  getproductPrice(){
if(product.equalsIgnoreCase(" Videogame ")) return 1000;
if(product.equalsIgnoreCase(" Water ")) return 100;
if(product.equalsIgnoreCase(" Meat ")) return 2000;
if(product.equalsIgnoreCase(" Bread ")) return 500;
return 0;
}


function getproductQuantity(){
let productQuality = (" How many do you want ")
while(!input.hasNextDouble()){
console.log(" invalid input : Try again ")
input.next()
return productQuality;
}
}


function getdateTime(){
return dateTime.now();


function getstoreReceipt( productName ,  customerName ,   cashierName , productQuantity ,  productPrice ){

let discount = productPrice * productQuantity * (7.5 / 100);
let vat = productPrice * productQuantity * (17.58 / 100);
let total = productPrice * productQuantity - discount + vat

console.log("\n====================================================");
console.log("\n  SEMICOLON STORE   ");
console.log("\n  MAIN BRANCH  " );
console.log("\n location: 312 HERBERT MACAULAY WAY SABO YABA LAGOS ");
console.log("\n Date : %s%n" , getdateTime());
console.log("\n Customer Name: %s%n ", customerName ); 
console.log("\n Cashier Name: %s%n ", cashierName );
console.log("\n=======================================================");
console.log("\n ITEM       QTY       PRICE       TOTAL(NGN)");
console.log("\n--------------------------------------------------------");
console.log("\n sub total : %2.f\nDISCOUNT: -%2.f \nVAT: +%2.f ",    (productPrice * productQuantity),  discount ,  vat );
console.log("\n BILL TOTAL: %2.f%n" ,  total);
console.log("\n===========================================================");
console.log("\n THIS IS NOT AN RECEIPT, KINDLY PAY "  ,   total );
console.log("\n=============================================================");
}


 main()
let productName = getproductName
let customerName = getcustomerName
let cashierName  = getcashierName
let  productQuantity = getproductQuantity
let  productPrice =  getproductPrice


if (productPrice == 0){
console.log("\n Invalid  product  selection : please restart");
return;

}
getstoreReceipt( productName ,  customerName ,   cashierName ,  productQuantity , productPrice )

}






































